tests/acceptance/conftest.py

```diff
@@ -86,6 +86,11 @@ def other_group(gl: Gitlab):
         gl.groups.delete(group_name)
 
 
+@pytest.fixture(scope="function")
+def random_string() -> str:
```

Comment:
Why do we need this if we already have another function for random string? We're calling that one in this function anyway.

Reply:
as far as I could tell, the other random_string function is not exposed on the scope for Tests to use @amimas

Reply:
Confirmed trying to use get_random_name directly results in:
```
fixture 'get_random_name' not found
```